src/dmtcpplugin.cpp

```diff
@@ -199,7 +199,10 @@ EXTERNC int dmtcp_set_ckpt_dir(const char* dir)
 EXTERNC const char* dmtcp_get_coord_ckpt_dir(void)
 {
   static string dir;
```

Comment:
Is there any reason we want a static string object here? If not, can we get rid of this static string object?